pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -344,17 +352,20 @@ func (l *LibvirtDomainManager) GuestPing(domainName string) error {
 }
 
 func getVMIEphemeralDisksTotalSize() *resource.Quantity {
-	var baseDir = "/var/run/kubevirt-ephemeral-disks/"
+	baseDir := getEphemeralDiskBaseDir()
 	totalSize := int64(0)
 	err := filepath.Walk(baseDir, func(path string, f os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
 		if !f.IsDir() {
 			totalSize += f.Size()
 		}
-		return err
+		return nil
 	})
 	if err != nil {
 		log.Log.Reason(err).Warning("failed to get VMI ephemeral disks size")
-		return &resource.Quantity{Format: resource.BinarySI}
+		return resource.NewScaledQuantity(totalSize, 0)
```

Comment:
I think it's better to be explicit and return a zero/uninitialized value here otherwise it would be impossible to tell whether the returned quantity represents a partial sum (thus an error) or the correct total size of disks as the Walk function might return an error anytime, even in the middle of the dirs traversal.

Reply:
Awesome you cached this. Fully agree. Thanks!